### ISSUE TITLE
CryptographicException when user doesn't have .NET 4.6.2 installed

### ISSUE THREAD

**Issue by wilksj** [Microsoft/WindowsTemplateStudio#591]
In the output windows I get the following: 
```

========== Tracked Exception [20170705 10:19:55.651] ==========
AdditionalMessage: The templates content can't be extracted.
System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(X509Certificate2 certificate, Boolean verifySignatureOnly)
   at MS.Internal.IO.Packaging.XmlDigitalSignatureProcessor.Verify(X509Certificate2 signer)
   at System.IO.Packaging.PackageDigitalSignature.Verify(X509Certificate signingCertificate)
   at System.IO.Packaging.PackageDigitalSignature.Verify()
   at System.IO.Packaging.PackageDigitalSignatureManager.VerifySignatures(Boolean exitOnFailure)
   at Microsoft.Templates.Core.Locations.Templatex.ValidateSignatures(Package package)
   at Microsoft.Templates.Core.Locations.Templatex.Extract(String signedFilePack, String targetDirectory, Boolean verifySignatures)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractContent(String file, String tempFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractMstx(String mstxFilePath)
   at Microsoft.Templates.Core.Locations.TemplatesSource.Extract(String mstxFilePath, String targetFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
--------------------------------------------------------------
[10:19:55.651 - Error]::Microsoft.Templates.Core.RepositorySynchronizationException: Error syncing templates. Status: 'Adquiring' ---> System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(X509Certificate2 certificate, Boolean verifySignatureOnly)
   at MS.Internal.IO.Packaging.XmlDigitalSignatureProcessor.Verify(X509Certificate2 signer)
   at System.IO.Packaging.PackageDigitalSignature.Verify(X509Certificate signingCertificate)
   at System.IO.Packaging.PackageDigitalSignature.Verify()
   at System.IO.Packaging.PackageDigitalSignatureManager.VerifySignatures(Boolean exitOnFailure)
   at Microsoft.Templates.Core.Locations.Templatex.ValidateSignatures(Package package)
   at Microsoft.Templates.Core.Locations.Templatex.Extract(String signedFilePack, String targetDirectory, Boolean verifySignatures)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractContent(String file, String tempFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractMstx(String mstxFilePath)
   at Microsoft.Templates.Core.Locations.TemplatesSource.Extract(String mstxFilePath, String targetFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   --- End of inner exception stack trace ---
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>b__24_0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<CheckMandatoryAdquisitionAsync>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<Do>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.TemplatesRepository.<SynchronizeAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.UI.ViewModels.MainViewModel.<InitializeAsync>d__77.MoveNext()
========== Tracked Exception [20170705 10:19:55.651] ==========
Microsoft.Templates.Core.RepositorySynchronizationException: Error syncing templates. Status: 'Adquiring' ---> System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(X509Certificate2 certificate, Boolean verifySignatureOnly)
   at MS.Internal.IO.Packaging.XmlDigitalSignatureProcessor.Verify(X509Certificate2 signer)
   at System.IO.Packaging.PackageDigitalSignature.Verify(X509Certificate signingCertificate)
   at System.IO.Packaging.PackageDigitalSignature.Verify()
   at System.IO.Packaging.PackageDigitalSignatureManager.VerifySignatures(Boolean exitOnFailure)
   at Microsoft.Templates.Core.Locations.Templatex.ValidateSignatures(Package package)
   at Microsoft.Templates.Core.Locations.Templatex.Extract(String signedFilePack, String targetDirectory, Boolean verifySignatures)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractContent(String file, String tempFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractMstx(String mstxFilePath)
   at Microsoft.Templates.Core.Locations.TemplatesSource.Extract(String mstxFilePath, String targetFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   --- End of inner exception stack trace ---
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>b__24_0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<CheckMandatoryAdquisitionAsync>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<Do>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.TemplatesRepository.<SynchronizeAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.UI.ViewModels.MainViewModel.<InitializeAsync>d__77.MoveNext()
--------------------------------------------------------------
```

**Comment by ralarcon** [comment]
Hi @wilksj, can you please let us know your environment configuration? Mainly:
* OS Version and patch Level
* Visual Studio Version and patch level
* .NET Framework versions installed
* Are you using the Windows Template Studio from the marketplace or a nightly / pre-release version?

Looks like the algorithm used to validate the packages signature is not available. I have found some references mentioning this:
https://blogs.msdn.microsoft.com/smondal/2012/08/24/signaturedescription-could-not-be-created-for-the-signature-algorithm-supplied/
https://stackoverflow.com/questions/16728558/signed-xml-signature-verification-for-sso-saml-using-sha256

**Comment by wilksj** [comment]
I'm running Windows 10 (10.0.10586), VS 2017 15.5 (26430.13). I also have VS2015 installed.

I ran the Windows Template Studio install from the Marketplace.

**Comment by ralarcon** [comment]
Is VS version 15.5 (26430.13) or 15 **.2** (26430.13) 

Then, can you make an screenshot to the properties of any app to see which frameworks do you have installed, something like:
![image](https://user-images.githubusercontent.com/2638875/27864319-5cbab906-618e-11e7-9e56-a68fbebc4d0b.png)

**Comment by 111rj** [comment]
I seem to be getting a similar error in the output window:

Windows Template Studio 2017-07-08 16:50:11.870
Version: 1.1.17173.2
>
[04:50:11.183 - Information]::Configuration file loaded: C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO\2017\COMMUNITY\COMMON7\IDE\EXTENSIONS\3ZSVAEFB.GSJ\WindowsTemplateStudio.config.json
[04:50:14.902 - Error]::Microsoft.Templates.Core.RepositorySynchronizationException: Error syncing templates. Status: 'Adquiring' ---> System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(X509Certificate2 certificate, Boolean verifySignatureOnly)
   at MS.Internal.IO.Packaging.XmlDigitalSignatureProcessor.Verify(X509Certificate2 signer)
   at System.IO.Packaging.PackageDigitalSignature.Verify(X509Certificate signingCertificate)
   at System.IO.Packaging.PackageDigitalSignature.Verify()
   at System.IO.Packaging.PackageDigitalSignatureManager.VerifySignatures(Boolean exitOnFailure)
   at Microsoft.Templates.Core.Locations.Templatex.ValidateSignatures(Package package)
   at Microsoft.Templates.Core.Locations.Templatex.Extract(String signedFilePack, String targetDirectory, Boolean verifySignatures)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractContent(String file, String tempFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractMstx(String mstxFilePath)
   at Microsoft.Templates.Core.Locations.TemplatesSource.Extract(String mstxFilePath, String targetFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   --- End of inner exception stack trace ---
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>b__24_0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<CheckMandatoryAdquisitionAsync>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<Do>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.TemplatesRepository.<SynchronizeAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.UI.ViewModels.MainViewModel.<InitializeAsync>d__77.MoveNext()
========== Tracked Exception [20170708 04:50:14.902] ==========
========== Tracked Exception [20170708 04:50:14.902] ==========
Microsoft.Templates.Core.RepositorySynchronizationException: Error syncing templates. Status: 'Adquiring' ---> System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(X509Certificate2 certificate, Boolean verifySignatureOnly)
   at MS.Internal.IO.Packaging.XmlDigitalSignatureProcessor.Verify(X509Certificate2 signer)
   at System.IO.Packaging.PackageDigitalSignature.Verify(X509Certificate signingCertificate)
   at System.IO.Packaging.PackageDigitalSignature.Verify()
   at System.IO.Packaging.PackageDigitalSignatureManager.VerifySignatures(Boolean exitOnFailure)
   at Microsoft.Templates.Core.Locations.Templatex.ValidateSignatures(Package package)
   at Microsoft.Templates.Core.Locations.Templatex.Extract(String signedFilePack, String targetDirectory, Boolean verifySignatures)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractContent(String file, String tempFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractMstx(String mstxFilePath)
   at Microsoft.Templates.Core.Locations.TemplatesSource.Extract(String mstxFilePath, String targetFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   --- End of inner exception stack trace ---
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>b__24_0()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<AdquireContentAsync>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<CheckMandatoryAdquisitionAsync>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.<Do>d__22.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.Core.TemplatesRepository.<SynchronizeAsync>d__17.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Templates.UI.ViewModels.MainViewModel.<InitializeAsync>d__77.MoveNext()
AdditionalMessage: The templates content can't be extracted.
--------------------------------------------------------------
System.Security.Cryptography.CryptographicException: SignatureDescription could not be created for the signature algorithm supplied.
   at System.Security.Cryptography.Xml.SignedXml.CheckSignedInfo(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(AsymmetricAlgorithm key)
   at System.Security.Cryptography.Xml.SignedXml.CheckSignature(X509Certificate2 certificate, Boolean verifySignatureOnly)
   at MS.Internal.IO.Packaging.XmlDigitalSignatureProcessor.Verify(X509Certificate2 signer)
   at System.IO.Packaging.PackageDigitalSignature.Verify(X509Certificate signingCertificate)
   at System.IO.Packaging.PackageDigitalSignature.Verify()
   at System.IO.Packaging.PackageDigitalSignatureManager.VerifySignatures(Boolean exitOnFailure)
   at Microsoft.Templates.Core.Locations.Templatex.ValidateSignatures(Package package)
   at Microsoft.Templates.Core.Locations.Templatex.Extract(String signedFilePack, String targetDirectory, Boolean verifySignatures)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractContent(String file, String tempFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSource.ExtractMstx(String mstxFilePath)
   at Microsoft.Templates.Core.Locations.TemplatesSource.Extract(String mstxFilePath, String targetFolder)
   at Microsoft.Templates.Core.Locations.TemplatesSynchronization.AdquireContent()
--------------------------------------------------------------

**Comment by 111rj** [comment]
Installed Components info : 


Microsoft Visual Studio Community 2017 
Version 15.2 (26430.15) Release
VisualStudio.15.Release/15.2.0+26430.15
Microsoft .NET Framework
Version 4.6.01038

Installed Version: Community

Visual Basic 2017   00369-60000-00001-AA309
Microsoft Visual Basic 2017

Visual C# 2017   00369-60000-00001-AA309
Microsoft Visual C# 2017

Visual C++ 2017   00369-60000-00001-AA309
Microsoft Visual C++ 2017

Application Insights Tools for Visual Studio Package   8.6.00404.2
Application Insights Tools for Visual Studio

ASP.NET and Web Tools 2017   15.0.30503.0
ASP.NET and Web Tools 2017

ASP.NET Web Frameworks and Tools 2017   5.2.50303.0
For additional information, visit https://www.asp.net/

Azure App Service Tools v3.0.0   15.0.30209.0
Azure App Service Tools v3.0.0

Common Azure Tools   1.9
Provides common services for use by Azure Mobile Services and Microsoft Azure Tools.

JavaScript Language Service   2.0
JavaScript Language Service

JavaScript Project System   2.0
JavaScript Project System

JavaScript UWP Project System   2.0
JavaScript UWP Project System

Microsoft Visual Studio VC Package   1.0
Microsoft Visual Studio VC Package

NuGet Package Manager   4.2.0
NuGet Package Manager in Visual Studio. For more information about NuGet, visit http://docs.nuget.org/.

Office Developer Tools for Visual Studio 2017 ENU   15.0.26309
Microsoft Office Developer Tools for Visual Studio 2017 ENU

SQL Server Data Tools   15.1.61702.140
Microsoft SQL Server Data Tools

TypeScript   2.2.2.0
TypeScript tools for Visual Studio

Visual Studio tools for CMake   1.0
Visual Studio tools for CMake

Visual Studio Tools for Universal Windows Apps   15.0.26430.06
The Visual Studio Tools for Universal Windows apps allow you to build a single universal app experience that can reach every device running Windows 10: phone, tablet, PC, and more. It includes the Microsoft Windows 10 Software Development Kit.

Workflow Manager Tools 1.0   1.0
This package contains the necessary Visual Studio integration components for Workflow Manager.

**Comment by crutkas** [comment]
I'm thinking, could a computer's clock (or some form of datetime) being off be causing this?  Look @111rj  logs, that is a 12 hr difference.

Windows Template Studio 2017-07-08 **16:50**:11.870
Version: 1.1.17173.2

[**04:50**:11.183

**Comment by mrlacey** [comment]
@crutkas time difference looks more like a time formatting issue where something is using "hh" for hours when it should be "HH". There's actually a few of them so have raised #674

**Comment by ralarcon** [comment]
Hello,

All the documentation related to this exception points to the lack of an algorithm.

At some point, we are using a SHA-2 algorithm to validate the Templates content downloaded / distributed have not being tampered. We use `SignedXml.XmlDsigSHA512Url` as hash algorithm.

```
            var dsm = new PackageDigitalSignatureManager(package)
            {
                CertificateOption = CertificateEmbeddingOption.InSignaturePart,
                HashAlgorithm = SignedXml.XmlDsigSHA512Url
            };
```

The support for SHA-2 alogorithms in xmlsignatures has been in introduced in .NET Fx 4.6.2 as per https://blogs.msdn.microsoft.com/dotnet/2016/03/30/announcing-the-net-framework-4-6-2-preview/

@111rj  You mention you are using .NET Fx 4.6.0... can you please install the .NET Fx 4.6.2, can you double check it by looking the list of framworks available (as I found the version shown in VS is not exacty the same...)

By the way, if you lack the .NET FX 4.6.2, you can pick it up from here: http://www.microsoft.com/en-us/download/details.aspx?id=53345.

**Comment by crutkas** [comment]
@wilksj and @111rj can either of you verify @ralarcon's theory?

**Comment by crutkas** [comment]
@wilksj and @111rj can you guys confirm this?

**Comment by wilksj** [comment]
I upgraded the framework to 4.6.2. After doing this I can confirm that I can get past the problem.

**Comment by crutkas** [comment]
Awesome!  Now to figure out why the installer didn't catch this.

**Comment by crutkas** [comment]
@wilksj thanks a ton for helping us track this issue down!

**Comment by crutkas** [comment]
brand new VM with VS 15.3 with UWP workload., office installed.  4.6.2 was not installed by default by VS installer for 15.3.  I could not load the Big.sln without this and VS error out.

I downloaded the WTS extension and it ran without a problem out of the gate.

Only difference could be is 15.3 just came out versus 15.2

**Comment by ralarcon** [comment]
The last behaviour that @crutkas mentions is addressed yet (x-ref: #967).

Can we close this as it looks that installing the minimum required framework solves the original issue?

**Comment by crutkas** [comment]
Lets close it and just keep an eye on the errors via telem
